src/webui/src/components/overview/config/TrialConfigPanel.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { Stack, Panel, Pivot, PivotItem, PrimaryButton } from '@fluentui/react';
+import { EXPERIMENT } from '../../../static/datamodel';
+import MonacoEditor from 'react-monaco-editor';
+import { MONACO } from '../../../static/const';
+import { convertDuration } from '../../../static/function';
+import { prettyStringify } from '../../../static/json_util';
+import lodash from 'lodash';
+import '../../../static/style/logDrawer.scss';
+
+interface LogDrawerProps {
+    hideConfigPanel: () => void;
+    activeTab?: string;
+}
+
+interface LogDrawerState {
+    panelInnerHeight: number;
+}
+
+class TrialConfigPanel extends React.Component<LogDrawerProps, LogDrawerState> {
+    constructor(props: LogDrawerProps) {
+        super(props);
+
+        this.state = {
+            panelInnerHeight: window.innerHeight
+        };
+    }
+
+    setLogDrawerHeight = (): void => {
+        this.setState(() => ({ panelInnerHeight: window.innerHeight }));
```

Comment:
Seems the arrow function is not necessary.

Reply:
fixed.